tools/workspace-plugin/src/generators/prepare-initial-release/lib/utils.ts

```diff
@@ -0,0 +1,43 @@
+import type { Tree } from '@nx/devkit';
+import ignore from 'ignore';
+import { join, relative, sep } from 'node:path';
+
+/**
+ * Utility to act on all files in a tree that are not ignored by git.
+ * copied from https://github.com/nrwl/nx/blob/master/packages/devkit/src/generators/visit-not-ignored-files.ts
+ * utility override from original in order to process files that are ignored within .nxignore
```

Comment:
we define `/bundle-size/` within `.nxignore` to not get circular dependencies by accident ( which have nothing to do with `library` implementation - thus we need custom visitor fn that won't take nxignore into consideration.